js/sum.js

```diff
@@ -0,0 +1,46 @@
+let addBtn = document.getElementById('add-btn');
+let newNum = document.getElementById('new-num');
+let numsContainer = document.getElementById('added-nums-container');
+let sumElem = document.getElementById('sum');
+let isOdd = false;
+
+addBtn.addEventListener('click', () => {
```

Comment:
This function contains side effects.

Reply:
You can specify what kind of side effects you see